code/modules/detective_work/evidence.dm

```diff
@@ -80,3 +80,175 @@
 	else
 		to_chat(user, "[src] is empty.")
 		icon_state = "evidenceobj"
+
+/obj/item/forensics
+	icon = 'icons/obj/forensics/forensics.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	new_attack_chain = TRUE
+
+/obj/item/sample
+	name = "\improper Forensic sample"
+	icon = 'icons/obj/forensics/forensics.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	var/list/evidence = list()
+	new_attack_chain = TRUE
```

Comment:
You have a bunch of sample code in random order, it goes

sample, print, sample, print, sample, fibers... etc. All the code for a single type should all be collected together and put the subtypes below.